src/pages/storage/StorageVolumes.tsx

```diff
@@ -402,20 +422,45 @@ const StorageVolumes: FC = () => {
           itemName="volume"
           className="u-no-margin--top"
           aria-label="Table pagination control"
+          description={
+            selectedNames.length > 0 && (
+              <SelectedTableNotification
+                totalCount={filteredVolumes.length ?? 0}
```

Comment:
```suggestion
                totalCount={volumes.filter((volume) => !isSnapshot(volume)).length}
```
This should be the unfiltered list, so we get "select all n filtered volumes" copy in case filters are applied:

![image](https://github.com/user-attachments/assets/395232ae-394d-4c1d-9e0c-ff3361ca881f)

Reply:
This should be the right value for it:

```
              <SelectedTableNotification
                totalCount={
                  volumes.filter(
                    (volume) =>
                      !isSnapshot(volume) && hasVolumeDetailPage(volume),
                  ).length
                }
                itemName="volume"
                parentName="project"
                selectedNames={selectedNames}
                setSelectedNames={setSelectedNames}
                filteredNames={filteredVolumes
                  .filter(hasVolumeDetailPage)
                  .map(getVolumeKey)}
              />
```